fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationSettings.kt

```diff
@@ -88,7 +89,10 @@ fun TranslationSettings(
                         modifier = Modifier
                             .fillMaxWidth()
                             .padding(start = 56.dp),
-                        onClick = { onAutomaticTranslationClicked() },
+                        onClick = {
+                            Events.translationsAction.record(Events.TranslationsActionExtra("global_lang_settings"))
```

Comment:
The same idea like here https://github.com/mozilla-mobile/firefox-android/pull/5942/files#r1517587430

Reply:
Updated!